for_challenges.py

```diff
@@ -40,7 +47,12 @@
     ['Вася', 'Маша', 'Саша', 'Женя'],
     ['Оля', 'Петя', 'Гриша'],
 ]
-# ???
+
+for group in enumerate(groups, start=1):
```

Comment:
распакуй же сразу: 
```suggestion
for group_number, group in enumerate(groups, start=1):
```
И в остальных местах тоже. Эта техника называется list unpacking, кода ты паре переменных присваиваешь значение листа.